plugin-viewport/src/main/kotlin/com/mapbox/maps/plugin/viewport/data/ViewportDataSourceProcessor.kt

```diff
@@ -0,0 +1,97 @@
+package com.mapbox.maps.plugin.viewport.data
+
+import com.mapbox.common.Logger
+import com.mapbox.maps.EdgeInsets
+import com.mapbox.maps.ScreenBox
+import com.mapbox.maps.ScreenCoordinate
+import com.mapbox.maps.Size
+
+internal object ViewportDataSourceProcessor {
```

Comment:
Add java doc for this object.

Reply:
This is an internal class, no documentation is required, but would add it later when we finalise the public interfaces.